jobs/geo/core_us.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+from jobs.utils.snowflake import gdf_to_snowflake, snowflake_connection_from_environment
+
+
+def load_us_geo_data(conn, year: str) -> None:
+    """Load U.S. Census geo data into Snowflake."""
+    from pygris import (
+        coastline,
+        divisions,
+        nation,
+        native_areas,
+        primary_roads,
+        rails,
+        regions,
+        states,
+        tribal_block_groups,
+        tribal_subdivisions_national,
+        urban_areas,
+    )
+
+    print(f"Downloading data for the U.S. in year {year}")
+
+    cache = True
+    dfs = []
+
+    nat = nation(year=year, cache=cache)
+    nat["year"] = year
+    dfs.append(nat)
+
+    division = divisions(year=year, cache=cache)
+    division["year"] = year
+    dfs.append(division)
+
+    region = regions(year=year, cache=cache)
+    region["year"] = year
+    dfs.append(region)
+
+    state = states(year=int(year), cache=cache)
+    state["year"] = year
+    dfs.append(state)
+
+    coastln = coastline(year=int(year), cache=cache)
+    coastln["year"] = year
+    dfs.append(coastln)
+
+    # core_based_statistical_area = core_based_statistical_areas(year=year, cache=cache)
+    # core_based_statistical_area['year'] = year
+    # dfs.append(core_based_statistical_area) #avaialble for 2021, but not 2022
+
+    # combined_statistical_area = combined_statistical_areas(year=year, cache=cache)
+    # combined_statistical_area['year'] = year
+    # dfs.append(combined_statistical_area) #avaialble for 2021, but not 2022
```

Comment:
This would be a good opportunity for a try/except. If `pygris` throws a recognizable error, you can catch it and just skip the table if it's not available for a year.

Reply:
Ha I thought of this last night while I was trying to sleep

Reply:
done